applications/osb-portal/src/assets/keycloak.json

```diff
@@ -0,0 +1,9 @@
+{
+  "realm": "ch",
+  "auth-server-url": "https://accounts.cloudharness.metacell.us/auth/",
```

Comment:
@zsinnema we need to make this address dynamic. Some ideas: use a resourceMap or environment variables from the backend

Reply:
The env variable $DOMAIN is available within all containers. Also need to fix in the workspace manager

Reply:
I've changed it in my last commit